docs/user-guide/README.md

```diff
@@ -69,3 +69,43 @@ Before submitting document change, you can run the same mdbook binary to preview
 
 You should have the user-guide available now at `localhost:3000`.
 Also, the serve command watches the `src` directory for changes and rebuilds the user-guide for every change.
+
+## Supported release versions
+
+Currently, in metal3 organization only CAPM3 and IPAM follow CAPI release cycles. The supported versions for CAPM3 and
+IPAM releases are as follows:
+
+- CAPM3
+    - v1beta1
+        - v1.1.1, v1.1.0
+    - v1alpha5
+        - v0.5.5, v0.5.4, v0.5.3, v0.5.2, v0.5.1, v0.5.0
+- IPAM
+    - v1alpha1
+        - v1.1.2, v1.1.1, v1.1.0, v0.1.2, v0.1.1, v0.1.0
+
+The compatability of IPAM and CAPM3 api versions with CAPI is discussed [here](https://github.com/metal3-io/ip-address-manager#compatibility-with-cluster-api).
+
+Since BMO and Ironic do not follow similar release cycles they are backwards compatible. However, we always tag the BMO and Ironic code base whenever we do a release in CAPM3. The tags have a prefix **CAPM3-** and the suffix is always the the corresponding CAPM3- release version. So for example if we cut a release for CAPM3 v1.0.0, we create a tag in the BMO and Ironic code base with CAPM3-v1.0.0. Following the same trend, the following tags for BMO and Ironic are available and supported (image tags):
```

Comment:
should we maybe be more precise and use lower case letters for CAPM3 prefix here and in the list, since that is how it looks exactly? 
```suggestion
Since BMO and Ironic do not follow similar release cycles they are backward compatible. However, we always tag the BMO and Ironic code base whenever we do a release in CAPM3. The tags have a prefix **CAPM3-** and the suffix is always the corresponding CAPM3- release version. So for example, if we cut a `v1.0.0` release for CAPM3 we create a tag in the BMO and Ironic code base with `capm3-v1.0.0`. Following the same trend, the following tags for BMO and Ironic are available and supported (image tags):
```